src/main/scala/scalaz/stream/writer.scala

```diff
@@ -0,0 +1,20 @@
+package  scalaz.stream
+
+import scalaz.\/._
+
+
+object writer {
+
+  /** Promote a `Process` to a `Writer` that writes nothing. */
+  def lift[F[_], A](p: Process[F, A]): Writer[F, Nothing, A] =
```

Comment:
@pchiusano see code right above. I might be switching the naming convention here. Should the `Writer` that "writes nothing" be actually called `liftO`. This actually makes more sense, since we're lifting a `Process[F, O]` (a `Process` with "outputs") into a `Writer` that outputs those outputs :) So I'll switch what I just pasted above, agreed?